tests/integration/Makefile

```diff
@@ -34,11 +34,17 @@ endif
 
 .PHONY : all
 ifdef S2N_NO_PQ
-all: tls13 client_endpoints dynamic_record old_s_client s_client s_server gnutls_cli gnutls_serv sslyze
+TESTS = tls13 client_endpoints dynamic_record old_s_client s_client s_server gnutls_cli gnutls_serv
 else
-all: tls13 client_endpoints dynamic_record old_s_client s_client s_server gnutls_cli gnutls_serv sslyze pq_handshake
+TESTS = tls13 client_endpoints dynamic_record old_s_client s_client s_server gnutls_cli gnutls_serv pq_handshake
 endif
 
+ifneq (S2N_NO_SSLYZE, "true")
+TESTS += sslyze
```

Comment:
If you're going to do it this way for sslyze, should you do it this way for S2N_NO_PQ too?

Reply:
We could, not seeing any scripts or Ci that use this flag.  [Readme](https://github.com/awslabs/s2n/blob/main/pq-crypto/README.md#how-to-disable-optimized-assembly-code-for-pq-crypto) will need updating and any downstream users might break.

Reply:
I meant add:
```
ifdef S2N_NO_PQ
TESTS += pq_handshake
endif
```
Instead of choosing between two long lists with and without pq_handshake. It seems messy to use two different mechanisms to select which tests are run.

Reply:
That does look cleaner, updated.